posthog/models/team/team.py

```diff
@@ -360,7 +360,7 @@ def groups_seen_so_far(self, group_type_index: GroupTypeIndex) -> int:
             {"team_id": self.pk, "group_type_index": group_type_index},
         )[0][0]
 
-    @cached_property
+    @property
```

Comment:
Hmm, why this change?

Reply:
It tripped up the tests. The test uses a team, asserts something, then sets the team timezone, and tries to assert once more. But due to the timezone property that the test uses being cached here on the instance, it didn't reflect the change. Not sure why it worked in the old tests.

But, I wouldn't recommend to put `cached_property` on "simple" properties like these anyways, since they are directly computed from available data and do not need to be queried for or such. It's often times more a source of confusion and stale data than helping performance. Hence I removed it. What do you think?